pkg/kubelet/cm/node_container_manager.go

```diff
@@ -58,12 +58,13 @@ func (cm *containerManagerImpl) createNodeAllocatableCgroups() error {
 // enforceNodeAllocatableCgroups enforce Node Allocatable Cgroup settings.
 func (cm *containerManagerImpl) enforceNodeAllocatableCgroups() error {
 	nc := cm.NodeConfig.NodeAllocatableConfig
+	enforce := !nc.EnforceNodeAllocatable.Has(kubetypes.NodeAllocatableNoneKey)
```

Comment:
Not sure this is strictly necessary, because in theory validation should require:
- only none
- empty
- subset of {pods, kube-reserved, system-reserved}

OTOH this is more explicit.

Reply:
I wouldn't change this, trust the validation preventing `none + node` on the way in

Reply:
updated